src/cards/Card.tsx

```diff
@@ -16,8 +16,11 @@ export const CardImage: React.FunctionComponent = ({children}) => {
     return <div className={ClassNames.CARD_IMAGE}>{children}</div>;
 };
 
-export const CardBlock: React.FunctionComponent = ({children}) => {
-    return <div className={ClassNames.CARD_BLOCK}>{children}</div>;
+export type CardBlockProps = {
```

Comment:
You don't need to export CardBlockProps. As currently you are using them in same file only.

Reply:
I agree here.

Reply:
Done.